process/block/metablock.go

```diff
@@ -933,7 +1229,6 @@ func (mp *metaProcessor) checkAndProcessShardMiniBlockHeader(
 }
 
 func (mp *metaProcessor) createShardInfo(
-	maxItemsInBlock uint32,
 	round uint64,
 	haveTime func() bool,
```

Comment:
haveTime is not used in this method, do we want this?

Reply:
yes. as we have haveTime in createBlockBody - header creation should not depend on time.

Reply:
Than you should remove this parameter as it is not used anywhere inside method

Reply:
will be removed in another pull request, it is already open.